inc/uds-blocks.php

```diff
@@ -80,3 +80,11 @@ function uds_wordpress_unregister_native_blocks() {
 
 	add_filter( 'allowed_block_types', 'uds_wordpress_unregister_native_blocks' );
 }
+
+/**
+ * Deregister core block patterns
+ */
+function remove_core_patterns() {
```

Comment:
I would add the line before starting the function: 
if ( ! function_exists( 'remove_core_patterns' ) )  
To avoid getting errors in case the function name was used in a child theme.